Streetcode/Streetcode.BLL/HealthChecks/LivenessHealthChecks.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Streetcode.BLL.HealthChecks.MemoryMetrics;
+
+namespace Streetcode.BLL.HealthChecks
+{
+    public class LivenessHealthChecks : IHealthCheck
+    {
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
+        {
+            var client = new MemoryMetricsClient();
+            var metrics = client.GetMetrics();
+            var percentUsed = 100 * metrics.Used / metrics.Total;
+
+            var status = HealthStatus.Healthy;
+            if (percentUsed > 80)
```

Comment:
You can put a block of assigning status to the private method for better readability. Also, in case percentUsed has a value more than 90 status variable will be overwritten 2 times. Consider optimizing this block of code

Reply:
I must assign a status anyways, because of error in result "Use of unassigned local variable 'status' ".